crates/edr_provider/src/data.rs

```diff
@@ -632,17 +633,6 @@ where
             RandomHashGenerator::with_seed("randomParentBeaconBlockRootSeed")
         };
 
-        let custom_precompiles = {
```

Comment:
Instead of having a boolean to indicate that we need to add a particular precompile, we now allow configuration of precompiles - making EDR more extensible.

The `enableRip7212` option was removed from HH3, instigating this change.